open_seq2seq/utils/utils.py

```diff
@@ -82,7 +82,7 @@ def clip_last_batch(last_batch, true_size):
   return last_batch_clipped
 
 
-def iterate_data(model, sess, compute_loss, mode, verbose):
+def iterate_data(model, sess, compute_loss, mode, verbose, input=None):
```

Comment:
I don't think it is useful to have input parameter here as well as in get_results_for_epoch, since these functions do much more than just pass one input through the model. But it would make more sense to create a separate function, something like run_model or get_model_output or something like that which will just accept the input and return model outputs.